src/rudderstack.ts

```diff
@@ -20,10 +22,20 @@ export class RudderStack {
         return RudderStack._instance
     }
 
-    /**
-     * @returns The anonymous ID assigned to the user before the identify event was called
-     */
-    getAnonymousId = () => this.analytics.getAnonymousId()
+    getAnonymousId = () => {
```

Comment:
can we add a condition here.
`if(this.analytics.getAnonymousId()!=undefined||this.analytics.getAnonymousId()!=''){
return this.analytics.getAnonymousId()
}
else{
 return document.cookie.match('(^|;)\\s*' + this.rudderstack_annonymous_cookie_key + '\\s*=\\s*([^;]+)')?.pop()
}`